pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -36,21 +36,41 @@ import (
 	"github.com/openshift/cluster-openshift-controller-manager-operator/pkg/util"
 )
 
-var knownControllers = []string{
-	"openshift.io/serviceaccount",
-	"openshift.io/default-rolebindings",
-	"openshift.io/serviceaccount-pull-secrets",
-	"openshift.io/origin-namespace",
-	"openshift.io/build",
-	"openshift.io/build-config-change",
-	"openshift.io/deployer",
-	"openshift.io/deploymentconfig",
-	"openshift.io/image-trigger",
-	"openshift.io/image-import",
-	"openshift.io/image-signature-import",
-	"openshift.io/templateinstance",
-	"openshift.io/templateinstancefinalizer",
-	"openshift.io/unidling",
+// All known Controllers
+var knownControllers = []controlplanev1.OpenShiftControllerName{
+	controlplanev1.OpenShiftServiceAccountController,
+	controlplanev1.OpenShiftDefaultRoleBindingsController,
+	controlplanev1.OpenShiftServiceAccountPullSecretsController,
+	controlplanev1.OpenshiftOriginNamespaceController,
+	controlplanev1.OpenshiftBuildController,
+	controlplanev1.OpenshiftBuildConfigChangeController,
+	controlplanev1.OpenshiftDeployerController,
+	controlplanev1.OpenshiftDeploymentConfigController,
+	controlplanev1.OpenshiftImageTriggerController,
+	controlplanev1.OpenshiftImageImportController,
+	controlplanev1.OpenshiftImageSignatureImportController,
+	controlplanev1.OpenshiftTemplateInstanceController,
+	controlplanev1.OpenshiftTemplateInstanceFinalizerController,
+	controlplanev1.OpenshiftUnidlingController,
+}
+
+// ControllerCapabilities maps controllers to capabilities, so we can enable/disable controllers
+// based on capabilities.
+var ControllerCapabilities = map[controlplanev1.OpenShiftControllerName]configv1.ClusterVersionCapability{
```

Comment:
No need to make this public. Only for internal purposes until it gets needed some place else.